content/kapp-controller/_index.html

```diff
@@ -55,108 +64,120 @@ <h2>Basic Usage</h2>
   serviceAccountName: default-ns-sa
   fetch:
   - git:
-      url: https://github.com/vmware-tanzu/carvel-simple-app-on-kubernetes
+      url: https://github.com/k14s/k8s-simple-app-example
```

Comment:
why did we revert this to k14s?

Reply:
This is what was under kapp-controller examples. But I didn't notice. Will change back.

Reply:
Cool, we should probably update that example too. I am happy to do that tomorrow